demo/vl-form-validation-e2e.html

```diff
@@ -0,0 +1,313 @@
+                <div class="vl-form-group">
+                    <div data-vl-validate-error-parent>
+                        <select name="validation-select" data-required="true"
+                            data-vl-success-class="success-style" data-vl-error-class="error-style"
+                            data-vl-error-message="Kies een stad" data-vl-validation-type="select"
+                            data-vl-error-placeholder="validation-select-error" aria-required="true" aria-invalid="true">
+                                <option placeholder value="">-- Kies een stad --</option>
+                                <option value="Aalst">Aalst</option>
+                                <option value="Antwerpen">Antwerpen</option>
+                            </select>
+                        </div>
+                        <div class="error-message" data-vl-error-id="validation-select-error"></div>
+                </div>
+            </form>
+        </div>
+    </div>
+
+    <div>
+        <h2>Verplichte validatie met succes melding</h2>
+        Gebruik het <strong>[data-vl-validate-form-success="true"]</strong> attribuut op de form om een succesvol ingevuld veld een stijl te geven.
+        <pre>
+            <code class="line-numbers language-markup">
+                &lt;form id="form-success-met-verplicht-veld" data-vl-validate-form data-vl-validate-form-success="true"&gt;
+                    &lt;label class="vl-form__label" for="validation-1"&gt;Naam&lt;/label&gt;
+                    &lt;div class="vl-input-group"&gt;
+                        &lt;input name="validation-naam" type="text" data-required="true"
+                            data-vl-success-class="success-style" data-vl-error-class="error-style"
+                            data-vl-error-message="Veld &quot;Naam&quot; is verplicht"
+                            data-vl-error-placeholder="validation-naam-error" aria-required="true" aria-invalid="true" /&gt;
+                    &lt;/div&gt;
+                    &lt;div class="error-message" data-vl-error-id="validation-naam-error"&gt;&lt;/div&gt;
+                &lt;/form&gt;
+            </code>
+        </pre>
+        <div class="demo">
+            <form id="form-success-met-verplicht-veld" data-vl-validate-form data-vl-validate-form-success="true">
+                <label class="vl-form__label" for="validation-1">Naam</label>
+                <div class="vl-input-group">
+                    <input name="validation-naam" type="text" data-required="true"
+                        data-vl-success-class="success-style" data-vl-error-class="error-style"
+                        data-vl-error-message="Veld &quot;Naam&quot; is verplicht"
+                        data-vl-error-placeholder="validation-naam-error" aria-required="true" aria-invalid="true" />
+                </div>
+                <div class="error-message" data-vl-error-id="validation-naam-error"></div>
+            </form>
+        </div>
+    </div>
```

Comment:
deze moet eigenlijk in een vl-ui-form zitten dus zou ik hier weghalen

Reply:
Moet deze naar de vl-ui-form? Ik kan niet goed inschatten wie (de form-validation component, of de form component) ervoor zorgt dat de succesklasse gezet wordt...

Reply:
het staat toch op form

Reply:
als je het in die context plaatst.... ja ;-)